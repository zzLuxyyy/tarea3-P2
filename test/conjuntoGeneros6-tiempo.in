conjuntoGenerosTiempo 1000000 1 
Fin