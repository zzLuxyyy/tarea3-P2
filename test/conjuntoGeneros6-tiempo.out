1>Prueba de conjunto generos (insertar, union, intersecciÃ³n, diferencia) para 1000000 entradas. Calculado correctamente en menos de 1s.
2>Fin.
